### DISCUSSION TITLE
`--include` should delete files by default

### DISCUSSION THREAD

**Discussion by emmyoop** [miniscruff/changie#261] [Ideas] [answered]
You replaced literally all the custom code I had written around changie to add footers and handle prereleases with ease!  Thank you for all your work!  I'm just hitting on (small) inconsistency in how I expect the workflow to behave.

In my workflow of having multiple prereleases before a final release, I move all my `yaml` files into a single version folder.

`changie batch <version>  --move-dir '<version>' --prerelease 'rc1'`
`changie batch <version>  --move-dir '<version>' --prerelease 'rc2`

Then when ready to do a final release I batch with 

`changie batch <version>  --include '<version>' --remove-prereleases`

This all works great except all of the `yaml` files in `.changes/<version>` persist and don't get deleted.  I would expect the `--include` flag to behave the same as using the default directory.  Is this intentional behavior?

**Comment by miniscruff** [comment, marked as answer]
Hmm that is strange, include should remove those files as well. I also think now it should remove the directory if it is empty. I will create two bugs and get them fixed. 

Edit: Fixed in v1.5.1
